jobrunner/cli/add_backend_to_job.py

```diff
@@ -0,0 +1,36 @@
+from jobrunner import config
+from jobrunner.lib.database import find_all, find_where, update
+from jobrunner.models import Job, SavedJobRequest
+
+
+def main():
+    """
+    Command to add missing backend attribute to jobs. We expect this to only
+    run once per backend, prior to moving the controller out of the backend
+    """
+    jobs = find_all(Job)
+    jobs_missing_backend = find_where(Job, backend=None)
+    if len(jobs) > len(jobs_missing_backend):
+        print("This command has already been run, please confirm you want to re-run:")
```

Comment:
Is this the case?  Once we've merged this PR, won't we start adding backends to jobs?  So unless we run this command immediately after merging and before we pick up any new jobs from job-server, there will be some jobs with backends, and so `jobs_missing_backends` won't be the same as `jobs`.  Or have I misunderstood?

Reply:
Hmm, yes, that is true. I just wanted for it to warn you if you might be re-running it with a jobs db that's not inside a backend (re-running it on an in-backend db won't do anything bad). I can re-word the warning; once it's been run on all the separate in-backend dbs, we can get rid of it anyway.

Reply:
I can skip the comparison actually, and just print a warning and require confirmation